src/main/java/gg/auroramc/collections/collection/Collection.java

```diff
@@ -142,6 +142,44 @@ private int getMultiplier(TypeId typeId, String trigger) {
         return multiplier;
     }
 
+    /**
+     * Displays various notifications (if enabled) to users chat
+     *
+     * @param player - the player to send the notification to
+     */
+    public synchronized void displayDiscoverMessage(Player player) {
+        List<Placeholder<?>> placeholders = List.of(
+                Placeholder.of("{collection_name}", this.config.getName())
+        );
```

Comment:
Use getPlaceholders() instead and add the raw name placeholder here as well, just like in levelup message.